app/views/editor/course/CourseEditView.coffee

```diff
@@ -0,0 +1,62 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/editor/course/edit'
+Course = require 'models/Course'
+ConfirmModal = require 'views/editor/modal/ConfirmModal'
+PatchesView = require 'views/editor/PatchesView'
+errors = require 'core/errors'
+app = require 'core/application'
+
+module.exports = class CourseEditView extends RootView
+  id: 'editor-course-edit-view'
+  template: template
+
+  events:
+    'click #save-button': 'onClickSaveButton'
+
+  constructor: (options, @courseID) ->
+    super options
+    @course = new Course(_id: @courseID)
+    @course.saveBackups = true
+    @supermodel.loadModel @course
+
+  onLoaded: ->
+    super()
+    @buildTreema()
+    @listenTo @course, 'change', =>
+      @course.updateI18NCoverage()
+      @treema.set('/', @course.attributes)
+
+  buildTreema: ->
+    return if @treema? or (not @course.loaded)
+    data = $.extend(true, {}, @course.attributes)
+    options =
+      data: data
+      filePath: "db/course/#{@course.get('_id')}"
+      schema: Course.schema
+      readOnly: me.get('anonymous')
+      supermodel: @supermodel
+    @treema = @$el.find('#course-treema').treema(options)
+    @treema.build()
+    @treema.childrenTreemas.rewards?.open(3)
+
+  afterRender: ->
+    super()
+    return unless @supermodel.finished()
+    @showReadOnly() if me.get('anonymous')
+    @patchesView = @insertSubView(new PatchesView(@course), @$el.find('.patches-view'))
```

Comment:
Any particular reason to use `.patches-view` instead of `#patches-view` here and in PatchesView.coffee (where the # version would presumably attach automatically)? Might there be multiple of them or something?